modules/cluster-logging-collector-log-forward-sts-cloudwatch.adoc

```diff
@@ -1,111 +1,65 @@
+* Create or update a `ClusterLogForwarder` custom resource (CR):
 +
-.CloudWatch credentials request template
 [source,yaml]
 ----
-apiVersion: cloudcredential.openshift.io/v1
-kind: CredentialsRequest
-metadata:
-  name: <your_role_name>-credrequest
-  namespace: openshift-cloud-credential-operator
-spec:
-  providerSpec:
-    apiVersion: cloudcredential.openshift.io/v1
-    kind: AWSProviderSpec
-    statementEntries:
-      - action:
-          - logs:PutLogEvents
-          - logs:CreateLogGroup
-          - logs:PutRetentionPolicy
-          - logs:CreateLogStream
-          - logs:DescribeLogGroups
-          - logs:DescribeLogStreams
-        effect: Allow
-        resource: arn:aws:logs:*:*:*
-  secretRef:
-    name: <your_role_name>
-    namespace: openshift-logging
-  serviceAccountNames:
-    - logcollector
-----
-+
-. Use the `ccoctl` command to create a role for AWS using your `CredentialsRequest` CR. With the `CredentialsRequest` object, this `ccoctl` command creates an IAM role with a trust policy that is tied to the specified OIDC identity provider, and a permissions policy that grants permissions to perform operations on CloudWatch resources. This command also creates a YAML configuration file in `/<path_to_ccoctl_output_dir>/manifests/openshift-logging-<your_role_name>-credentials.yaml`. This secret file contains the `role_arn` key/value used during authentication with the AWS IAM identity provider.
-+
-[source,terminal]
-----
-$ ccoctl aws create-iam-roles \
---name=<name> \
---region=<aws_region> \
---credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \
---identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com <1>
-----
-<1> <name> is the name used to tag your cloud resources and should match the name used during your STS cluster install
-+
-. Apply the secret created:
-[source,terminal]
-+
-----
-$ oc apply -f output/manifests/openshift-logging-<your_role_name>-credentials.yaml
-----
-+
-. Create or edit a `ClusterLogForwarder` custom resource:
-+
-[source,yaml]
-----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: openshift-logging
 spec:
-  serviceAccountName: clf-collector <3>
+  serviceAccountName:
+    name: <service_account_name> #<1>
   outputs:
-   - name: cw <4>
-     type: cloudwatch <5>
+   - name: cw-output #<2>
+     type: cloudwatch #<3>
      cloudwatch:
-       groupBy: logType <6>
-       groupPrefix: <group prefix> <7>
-       region: us-east-2 <8>
-     secret:
-        name: <your_secret_name> <9>
+       groupName: 'cw-projected{.log_type||"missing"}' #<4>
+       region: us-east-2 #<5>
+       authentication:
+         type: iamRole #<6>
+         iamRole:
+           roleARN: #<7>
+             key: credentials 
```

Comment:
I suggest to use 
```suggestion
             key: role_arn 
```
per the command to create the secret.

Reply:
done